<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWIndow</class>
 <widget class="QMainWindow" name="MainWIndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>728</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>ContactFEM</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(200, 200, 200);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,2,0,1,0,1" columnstretch="1,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="3" column="0" rowspan="3">
       <widget class="PlotWidget" name="graphicsView_Scheme_deformed"/>
      </item>
      <item row="3" column="1">
       <widget class="PlotWidget" name="graphicsView_contact_info_normal"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_contact_info_tangent">
        <property name="text">
         <string>Contact info tangent</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_scheme">
        <property name="text">
         <string>Scheme</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="PlotWidget" name="graphicsView_Scheme"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_scheme_deformed_unilateral_contact">
        <property name="text">
         <string>Scheme deformed unilateral contact</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="PlotWidget" name="graphicsView_Scheme_unilateral_contact"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_scheme_deformed_linear">
        <property name="text">
         <string>Scheme deformed linear</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_contact_info_normal">
        <property name="text">
         <string>Contact info normal</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="PlotWidget" name="graphicsView_contact_info_tangent"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad_Example">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="title">
      <string>Load Example</string>
     </property>
     <addaction name="actionSymmetric_detach"/>
     <addaction name="actionSimple_beam"/>
    </widget>
    <addaction name="actionRun"/>
    <addaction name="separator"/>
    <addaction name="menuLoad_Example"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_data_to_excel"/>
   </widget>
   <widget class="QMenu" name="menuShow_Hide">
    <property name="title">
     <string>Show/Hide</string>
    </property>
    <addaction name="actionNumerical"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionDeformation_Scale"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menuShow_Hide"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="separator"/>
   <addaction name="actionNumerical"/>
   <addaction name="actionDeformation_Scale"/>
  </widget>
  <action name="actionSymmetric_detach">
   <property name="text">
    <string>Symmetric detach</string>
   </property>
  </action>
  <action name="actionSimple_beam">
   <property name="text">
    <string>Simple beam</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save problem</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_data_to_excel">
   <property name="text">
    <string>Save data to excel</string>
   </property>
  </action>
  <action name="actionNumerical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Numerical</string>
   </property>
   <property name="toolTip">
    <string>Numericals on graphs</string>
   </property>
   <property name="shortcut">
    <string>Shift+N</string>
   </property>
  </action>
  <action name="actionDeformation_Scale">
   <property name="text">
    <string>Deformation Scale</string>
   </property>
   <property name="shortcut">
    <string>Shift+D</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
